tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -312,42 +310,36 @@ private Grid GetErrorCard(string error, string subError = null)
         return grid;
     }
 
-    private JObject WrapJsonString(string jsonString)
+    private Newtonsoft.Json.Linq.JObject WrapJsonString(string jsonString)

```

Comment:
Does `using Newtonsoft.Json.Linq` throw an error?

Reply:
Nope, that's just the namespace JObject lives under.